Gemfile.lock

```diff
@@ -168,22 +166,19 @@ GEM
       winrm (~> 2.0)
       winrm-elevated (~> 1.0)
       winrm-fs (~> 1.0)
-    chef-provisioning-aws (2.2.2)
+    chef-provisioning-aws (3.0.0.pre.rc1)
       aws-sdk (>= 2.2.18, < 3.0)
-      aws-sdk-v1 (>= 1.59.0)
       chef-provisioning (>= 1.0, < 3.0)
       retryable (~> 2.0, >= 2.0.1)
       ubuntu_ami (~> 0.4, >= 0.4.1)
-    chef-provisioning-azure (0.6.0)
+    chef-provisioning-fog (0.25.1)
       chef-provisioning (>= 1.0, < 3.0)
-      stuartpreston-azure-sdk-for-ruby (~> 0.7)
-    chef-provisioning-fog (0.21.0)
-      chef-provisioning (>= 1.0, < 3.0)
-      fog (>= 1.37.0)
+      cheffish (>= 5.0, < 14.0)
+      fog
       fog-digitalocean
       fog-scaleway
-      fog-softlayer (~> 1.1.0)
-      google-api-client (~> 0.8.0)
```

Comment:
this is still busted.

Reply:
how so?

Reply:
@lamont-granquist @thommay I'm able to build the DK off of Thom's commit with no errors and run test-kitchen successfully with the bundled `gce` driver.

```
~/Devel/ChefProject/chef-dk/omnibus ((c7c7ccb4...) *)$ git log --pretty=oneline --abbrev-commit -n 3
c7c7ccb4 (HEAD, origin/tm/update_provisioning) downgrade chef until test problems are fixed
3f935985 Dependency updates
8432fb6c (tag: v2.3.14, origin/master, origin/HEAD, master) Bump version to 2.3.14 by Chef Expeditor
~/Devel/ChefProject/chef-dk/omnibus ((c7c7ccb4...) *)$ chef -v
Chef Development Kit Version: 2.4.0
chef-client version: 13.3.42
delivery version: master (73ebb72a6c42b3d2ff5370c476be800fee7e5427)
berks version: 6.3.1
kitchen version: 1.18.0
inspec version: 1.41.0
~/Devel/ChefProject/chef-dk/omnibus ((c7c7ccb4...) *)$
```

I didn't come across any busted parts.
Can we forge ahead with merging this?

Reply:
chef-provisioning-fog is busted on master (the version that got pulled in here) against cheffish 5.0/13.0
fog-google is also known to be broken with the google-api-client that this change pulls in (even though no tests anywhere go red for this)

Reply:
fog-google issues:  https://github.com/fog/fog-google/pull/210 and https://github.com/fog/fog-google/pull/100